### ISSUE TITLE
Automerge doesn't appear for incoming pull requests

### ISSUE THREAD

**Issue by dstockwell** [PolymerLabs/project-health#535]
## Expected Behavior

After I approve an incoming pull request I should be able to set the auto-merge option from project-health.

**Comment by samuelli** [comment]
Are there any restrictions? Sample restrictions:
 - You must have merge abilities
 - PR author doesn't have merge abilities

**Comment by dstockwell** [comment]
> * PR author doesn't have merge abilities

Not sure about that one. That seems a case where the reviewer would be required to merge it.

**Comment by samuelli** [comment]
Right, but if they did have merge abilities, should you also see the auto-merge option? If you did, then both of you could have the auto merge option available for the same PR.

**Comment by dstockwell** [comment]
> Right, but if they did have merge abilities, should you also see the auto-merge option? If you did, then both of you could have the auto merge option available for the same PR.

Yes. I often merge other project member's PRs -- when we're operating in different timezones it results in changes landing much faster.

**Comment by samuelli** [comment]
> If both author + reviewer use automerge, that seems OK -- doesn't really matter too much which one goes ahead.

You could have contention over what method to use though.

@gauntface: how possible do you think this is? I'm not sure how the auto merge stuff is stored rn.

**Comment by gauntface** [comment]
Steps for adding this would be:

1.) Figure out when to show the UI for automerge to the reviewer.
2.) Store the automerge option in the backend (this is done against the PR itself, so would sync between reviewer and PR author).
3.) The automerge should just work from this.
